### ISSUE TITLE
Screen reader does not work on Linux Steam build

### ISSUE THREAD

**Issue by jonko0493** [ITDS-Project/itds-wrapper#75]
Changing the setting does not refresh the screen reader availability; it only activates on restart.

**Comment by jonko0493** [comment]
This actually does not seem to be the case on Linux. Needs testing on other platforms to confirm.

**Comment by jonko0493** [comment]
Adjusted the title for the actual issue. It doesn't work on the Steam build, but does work locally.

**Comment by jonko0493** [comment]
Nope, not fixed after last attempt.

**Comment by jonko0493** [comment]
Still not quite there. Most likely there's a couple of other libs we have to package and/or we should do a musl build?
